### DISCUSSION TITLE
isSuccess status only for Refetch

### DISCUSSION THREAD

**Discussion by StormLx** [TanStack/query#7470] [Q&A]
Hello !
I've seen different statuses for a query, such as "isSuccess" "isFetched" etc.. 
But what I want is the status "isRefetched" or/and "isRefetchedSuccess" but i dont see it.

nb: I'm using the experimental Angular V5

Any  idea?

**Comment by TkDodo** [comment]
those don't exist:

https://tanstack.com/query/v5/docs/framework/react/reference/useQuery
